README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://scrutinizer-ci.com/g/jbcrestot/NavitiaPhp/badges/build.png?b=master)](https://scrutinizer-ci.com/g/jbcrestot/NavitiaPhp/build-status/master)
```

Comment:
I think the PHPUnit tests are not run here, maybe we should add the rating badge here, and use travis for tests/PSR2 check